src/pages/projects/form/ContributorRow.tsx

```diff
@@ -89,8 +89,8 @@ export const ContributorRow = ({
       </TableCell>
       <TableCell>
         <>
-          {contributor.roles
-            .filter((role) => role.affiliation?.id)
+          {roles
+            .filter((role) => role.affiliation && role.affiliation.id)
```

Comment:
Ikke at det er noen forskjell så whatever 🤷 litt tvangstanker fra meg som må ut bare 😅 
```suggestion
            .filter((role) => role.affiliation?.id)
```